src/buildHTML.js

```diff
@@ -174,6 +200,21 @@ export const getTypeOfDomTree = function(node) {
     return null;
 };
 
+// If `node` is an atom return whether it's been assigned the mtight class.
+// If `node` is a document fragment, return the value of isLeftTight() for the
+// leftmost node in the fragment.
+// 'mtight' indicates that the node is script or scriptscript style.
+export const isLeftTight = function(node) {
+    if (node instanceof domTree.documentFragment) {
```

Comment:
Can the `node` be a `domTree.documentFragment`? Aren't they flattened at the start of `buildExpression`?

Reply:
Good catch.

Reply:
@kevinbarabash It seems that `node`'s children can be a `domTree.documentFragment`, e.g., `\phantom{x}^2`.